README.md

```diff
@@ -35,7 +31,50 @@ docker compose exec driver rake validate_all
 
 Output from the command will appear in your terminal.
 
-## Development Mode
+## Terminology
+
+The production UK federation tooling performs *checks* on metadata.
+
+The purpose of this testbed is to perform *tests* on those checks.
+
+Each test is composed of an XML *test file* and an optional
+sidecar *options file* controlling the execution of the test.
+
+A test is executed by sending the *test data* from the test file to
+one or more *endpoints* providing a metadata validation web service.
+
+Each addressed web service runs a requested *validator* on the test data.
+This is a named sequence of MDA stages; the result is a possibly empty
+set of *status metadata* which is returned to the testbed driver.
+
+A test *succeeds* if the returned status metadata matches the
+test's expected results. The test *fails* if any differences
+are detected.
+
+## Endpoints
+
+The `docker compose` deployment currently includes four containers
+implementing metadata validation web services which are referenced
+inside the testbed as the following named *endpoints*:
+
+- `v09x` represents the current production deployment. Its name reflects
+  that environment's composition: v0.9 of the Shibboleth Metadata
+  Aggregator, and the Xalan XSL processor included in the classpath.
+- `v09` is identical to `v09x` with the exclusion of the Xalan processor.
+  Because this can run neither the old nor new versions of the URL checking
+  code (the old system involves a Xalan extension, while the new system only
```

Comment:
I understand this, but I was just wondering if there was a link somewhere which explained what that URL checking code was, or a ticket—something for reference. Although, I guess it is unlikely somebody without the required context would need to understand this.

Reply:
It was my thinking that this was indeed too "inside baseball" to be worth expanding. Most of the documentation around this issue is in the private version of the `ukf-meta` repository, so it's also difficult to link to.

Reply:
Yup. That makes sense.